moviesurfer/cli/main.py

```diff
@@ -1,2 +1,28 @@
-def main():
-    print("Hello from CLI, functionality is to be added")
+import logging
+
+import click
+
+from moviesurfer.data.download_data import download_data
+
+logger = logging.getLogger(__name__)
+
+
+@click.group(chain=True)
+def cli():
+    logger.info("moviesurfer cli started")
+
+
+@cli.command()
+@click.argument("download")
+def data(download):
+    """Runs data processing scripts to turn raw data from (../raw) into
+    cleaned data ready to be analyzed (saved in ../processed).
+    """
+    try:
+        download_data()
+    except FileExistsError:
+        logger.error("No need to call 'download data', file exists")
```

Comment:
please, also drop a message to console using `click.echo('your message here')`

Reply:
Hm, I see, you use `logger.error`, that's why it prints it, but it is more of a side effect. If we decide to stream all logs to a file, it will disappear. I would not rely on logging to print CLI messages. Some messages may be just "info" level, for those this way won't work and we will have to use `click.echo`. We will end up with several ways to do the same thing which will be confusing.